styles/layout.less

```diff
@@ -154,6 +154,14 @@ footer, #footer {
     display: table-cell;
 }
 
+.quick-links {
```

Comment:
I was rather thinking something like this:
CSS:
```css
    ul.quick-links {
        padding: 0;
    }
    ul.quick-links li {
        display: inline;
    }
    ul.quick-links li::before {
        content: " | ";
    }
    ul.quick-links li:first-child::before {
        content: "";
    }
```
HTML:
```html
    <ul class='quick-links'>
        <li><a href='blah'>Heading 1</a></li>
        <li><a href='blah'>Heading 2</a></li>
        <li><a href='blah'>Heading 3</a></li>
    </ul>
    <h1>Heading 1</h1>
```
That way we can style and the separator (which is really only textual styling) independent of the code.

Reply:
I like this better. I usually avoid ::before due to accessibility concerns, but here it is something we don't want screen readers reading anyways.